dnase/callsnpsMerge.sh

```diff
@@ -142,6 +142,7 @@ case "${numfullbcffiles}" in
 *)
     bcftools concat --threads $NSLOTS --output-type b ${fullbcffiles} > ${sampleOutdir}/${name}.${mappedgenome}.bcf;;
 esac
+# FIXME: Should the .csi files be removed after merging the byChrom bcf's?
```

Comment:
Which .csi files are you asking about? If I look at a successful run, I only see a .csi files for the merged data, not the individual byChrom data.

Looking at the bcftools docs, it looks like we are creating redundant indices -- is that what you noticed? We should probably just switch to "bcftools index --tbi" which eliminates the .csi index and the subsequent tabix step. You can roll this into the same issue as your bgzip output question.

Reply:
It did not look like the csi files were being removed during the merge step. These were still in the directory.
I still have all those files with the job numbers in the file names in my output directory.

I was wondering about the indexing, but it was not something I looked into

Reply:
Opened #263

Reply:
You are right, the .bcf.csi files are never cleaned up. I will also add this to #263